components/aws/sagemaker/tests/integration_tests/component_tests/test_train_component.py

```diff
@@ -13,7 +13,14 @@
         pytest.param(
             "resources/config/simple-mnist-training", marks=pytest.mark.canary_test
         ),
-        pytest.param("resources/config/fsx-mnist-training", marks=pytest.mark.fsx_test),
+        pytest.param(
+            "resources/config/xgboost-mnist-trainingjob-debugger",
+            marks=pytest.mark.canary_test
```

Comment:
@akartsky Is this considered essential for a canary test? I would say probably not because it's testing a feature rather than a core functionality, but let's double check against the operator canaries.

Reply:
ya operator canaries do run debugger test

Reply:
@RedbackThomson are operator canaries running all tests in canaries ?

Reply:
Operators don't run every test, no. For training we run a basic mnist training job, and a debugger xgboost training job. This covers our basis for 2 algorithms and the debugger feature.